pkg/apis/pipeline/v1beta1/pipelinerun_validation_test.go

```diff
@@ -125,7 +125,7 @@ func TestPipelineRun_Invalidate(t *testing.T) {
 		t.Run(ps.name, func(t *testing.T) {
 			err := ps.pr.Validate(context.Background())
 			if d := cmp.Diff(err.Error(), ps.want.Error()); d != "" {
-				t.Errorf("PipelineRun.Validate/%s %s", ps.name, diff.PrintWantGot(d))
+				t.Error(diff.PrintWantGot(d))
```

Comment:
nit: got/want needs to be flipped to want/got to make this error msg correct 🤔

Reply:
oh on the `cmd.Diff`, indeed :sweat_smile: